app/models/host/discovered.rb

```diff
@@ -2,8 +2,7 @@
 
 class Host::Discovered < ::Host::Base
   include ScopedSearchExtensions
-
-  belongs_to :subnet
+  
```

Comment:
Can can you remove this hunk (leave lines 4-6 untouched in this patch). I am fixing this in a different patch.